src/test/java/io/confluent/examples/streams/window/CustomWindowTest.java

```diff
@@ -61,220 +61,243 @@
 
     @Test
     public void shouldSumNumbersOnSameDay() {
-        final List<MyEvent> inputValues = Arrays.asList(
-                new MyEvent(1, ZonedDateTime.of(2019, 1, 1, 16, 29, 0, 0, zone)),
-                new MyEvent(2, ZonedDateTime.of(2019, 1, 1, 16, 30, 0, 0, zone)),
-                new MyEvent(7, ZonedDateTime.of(2019, 1, 1, 16, 31, 0, 0, zone)),
-                dummyEventToForceSuppression()
-        );
-        final List<ExpectedResult> expectedValues = Arrays.asList(
-                new ExpectedResult(1,
-                        10,
-                        ZonedDateTime.of(2018, 12, 31, 18, 0, 0, 0, zone),
-                        ZonedDateTime.of(2019, 1, 1, 18, 0, 0, 0, zone))
+      final List<TestRecord<String, Integer>> inputValues = Arrays.asList(
+        new TestRecord<>(null,
+                         1,
+                         ZonedDateTime.of(2019, 1, 1, 16, 29, 0, 0, zone).toInstant()),
+        new TestRecord<>(null,
+                         2,
+                         ZonedDateTime.of(2019, 1, 1, 16, 30, 0, 0, zone).toInstant()),
+        new TestRecord<>(null,
+                         7,
+                         ZonedDateTime.of(2019, 1, 1, 16, 31, 0, 0, zone).toInstant()),
+        dummyEventToForceSuppression()
         );
-        verify(inputValues, expectedValues, zone);
+      final List<KeyValue<Windowed<Integer>, Integer>> expectedValues =
+        Collections.singletonList(KeyValue.pair(toWindowed(1,
```

Comment:
Using `toWindowed` to get `Windowed<K>` for the key passing in key, start/end ts